docs/providers/aws/guide/resources.md

```diff
@@ -81,4 +81,5 @@ We're also using the term `normalizedName` or similar terms in this guide. This
 |ApiGateway::Deployment | ApiGatewayDeployment{randomNumber}                      | ApiGatewayDeployment12356789  |
 |ApiGateway::ApiKey     | ApiGatewayApiKey{SequentialID}                          | ApiGatewayApiKey1             |
 |SNS::Topic             | SNSTopic{normalizedTopicName}                           | SNSTopicSometopic             |
+|SNS::Subscription      | {normalizedFunctionName}SnsSubscription{normalizedTopicName}   | HelloSnsSubscriptionSomeTopic             |
```

Comment:
hmmm "what's the use case" @pmuens very good question actually :) ... haven't thought about that. I guess no one really would realistically do that, since it'd end up being the same subscription anyway.

Ok let's ignore that.